majasdarbs/src/io/codelex/classesandobjects/practice/car/Car.java

```diff
@@ -0,0 +1,36 @@
+package io.codelex.classesandobjects.practice.car;
+
+public class Car {
+
+    public double startKilometers; // Starting odometer reading
+    public double endKilometers; // Ending odometer reading
+    public double liters; // Liters of gas used between the readings
+
+    public Car(double startOdo) {
+        this.startKilometers = startOdo;
+        this.endKilometers = startKilometers;
+        this.liters = 0;
+
+    }
+
+    void fillUp(int mileage, double liters) {
+        this.endKilometers = mileage;
+        this.liters = liters;
+    }
+
+    public double calculateConsumption() {
+
+        return liters / ((endKilometers - startKilometers) / 100);
+
+    }
+
+    public boolean gasHog() {
+        return calculateConsumption() > 15.0;
```

Comment:
konstante

Reply:
Nesapratu īsti.

Reply:
ka 15 definēt kā konstanti piemēram
`private static final double HIGH_FUEL_CONSUMPTION = 15.0`
in general tu vienmēr gribi definēt visus ciparus, lai nav magic numbers

Reply:
Paldies :)